.env.example

```diff
@@ -25,6 +25,8 @@ LOG_LEVEL=debug
 NOTIFICATIONS_EMAIL_USER="no-reply@senecacollege.ca"
 SMTP_PORT=1025
 
+NOTIFICATIONS_REPEAT_PATTERN=5 * 60 * 1000
```

Comment:
I don't think this needs to be configurable.  I would hard-code it into the app.